README.md

```diff
@@ -1,37 +1,48 @@
-# lua-resty-expr
+# Name
 
-## Name
+A tiny DSL to evaluate expressions inside [Apache APISIX](https://github.com/apache/apisix).
 
-A tiny DSL to evaluate expressions which is used inside of APISIX.
+# Status
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/api7/lua-resty-expr/blob/main/LICENSE)
 
-This project has been working in microservices API gateway [Apache APISIX](https://github.com/apache/incubator-apisix).
+Used by:
 
-The project is open sourced by [Shenzhen ZhiLiu](https://www.apiseven.com/) Technology Co., Ltd.
+- [Apache APISIX](https://github.com/apache/apisix): A high-performance cloud native API gateway.
 
-In addition to this open source version, our company also provides a more powerful and performing commercial version, and provides technical support. If you are interested in our commercial version, please [contact us](https://www.apiseven.com/).
+Developed by [API7.ai](https://api7.ai/).
 
-Table of Contents
-=================
+> **Note**
+>
+> API7.ai provides technical support for the software it maintains like this library and [Apache APISIX](https://github.com/apache/apisix). Please [contact us](https://api7.ai/contact) to learn more.
 
-* [lua-resty-expr](#lua-resty-expr)
-    * [Name](#name)
-    * [Synopsis](#synopsis)
-    * [Methods](#methods)
-        * [new](#new)
-            * [Comparison Operators](#comparison-operators)
-            * [Logical Operators](#logical-operators)
-        * [eval](#eval)
-    * [Install](#install)
-        * [Compile and install](#compile-and-install)
-    * [DEV ENV](#dev-env)
-        * [Install Dependencies](#install-dependencies)
+# Table of Contents
 
-## Synopsis
+- [Name](#name)
+- [Status](#status)
+- [Table of Contents](#table-of-contents)
+- [Synopsis](#synopsis)
+- [Methods](#methods)
+  - [new](#new)
+    - [Usage](#usage)
+  - [eval](#eval)
+    - [Usage](#usage-1)
+    - [Example](#example)
+- [Operators](#operators)
+  - [Comparison Operators](#comparison-operators)
+  - [Logical Operators](#logical-operators)
+  - [Example](#example-1)
+- [Installation](#installation)
+  - [From LuaRocks](#from-luarocks)
+  - [From Source](#from-source)
+- [Development](#development)
+
+# Synopsis
 
 ```lua
  location / {
+     set $arg_name 'json';
```

Comment:
I see there have been some code changes. I'm not available to check such changes 🙏

Reply:
I made this code change because a user can just copy paste it in his Nginx configuration and it would work. Just added setting an argument,